src/handlers/motions/motionRewardClaimed/helpers.ts

```diff
@@ -1,11 +1,44 @@
-import { mutate } from '~amplifyClient';
+import { BigNumber } from 'ethers';
+import { mutate, query } from '~amplifyClient';
 import {
+  GetColonyStakeDocument,
+  GetColonyStakeQuery,
+  GetColonyStakeQueryVariables,
   StakerReward,
   UpdateColonyDocument,
   UpdateColonyMutation,
   UpdateColonyMutationVariables,
+  UpdateColonyStakeDocument,
+  UpdateColonyStakeMutation,
+  UpdateColonyStakeMutationVariables,
+  UserStakes,
 } from '~graphql';
-import { getColonyFromDB } from '~utils';
+import { getColonyFromDB, verbose } from '~utils';
+
+import { getColonyStakeId } from '../helpers';
+
+export const getUpdatedStakerRewards = (
```

Comment:
This was previously in motionRewardClaimed.ts, moved here to keep that file neat